libs/core/src/lib/shellbar/user-actions-menu/user-actions-menu/user-actions-menu.component.ts

```diff
@@ -0,0 +1,105 @@
+import { ChangeDetectionStrategy, Component, ContentChildren, EventEmitter, Input, Output, QueryList, ViewChild, ViewEncapsulation } from '@angular/core';
+import { Placement, PopperOptions } from 'popper.js';
```

Comment:
As far as I know we a moving away from popper.js https://github.com/SAP/fundamental-ngx/issues/2527
@JKMarkowski could you help with it?

Reply:
Feel free to just import it from `libs/core/src/lib/popover/popover-position/popover-position.ts`, instead of having popper.js ref